plugins/notifications-backend/src/service/router.ts

```diff
@@ -102,7 +103,7 @@ export async function createRouter(
     const entities = await catalogClient.getEntitiesByRefs(
       {
         entityRefs: refs,
-        fields: ['kind', 'metadata.name', 'metadata.namespace'],
+        fields: ['kind', 'metadata.name', 'metadata.namespace', 'relations'],
```

Comment:
Requires `spec.owner` too as it's used here: https://github.com/backstage/backstage/pull/23990/files#diff-e2b0532e3202f4ab3b59bf9abd868e81fbe500bddf84148c467487447eb1b420R148

Reply:
Updated